From: Stephan Mueller <smueller@chronox.de>
Subject: crypto: drbg - select SHA512
References: bsc#1189034
Patch-mainline: Queued in subsystem maintainer repo
Git-repo: https://git.kernel.org/pub/scm/linux/kernel/git/herbert/cryptodev-2.6.git
Git-commit: 5261cdf457ce3635bf18d393a3c1991dcfaf9d02

With the swtich to use HMAC(SHA-512) as the default DRBG type, the
configuration must now also select SHA-512.

Fixes: 9b7b94683a9b "crypto: DRBG - switch to HMAC SHA512 DRBG as default
DRBG"
Reported-by: Sachin Sant <sachinp@linux.vnet.ibm.com>
Signed-off-by: Stephan Mueller <smueller@chronox.com>
Tested-by: Sachin Sant <sachinp@linux.vnet.ibm.com>
Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
Acked-by: Jeff Mahoney <jeffm@suse.com>
---
 crypto/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

(limited to 'crypto/Kconfig')

diff --git a/crypto/Kconfig b/crypto/Kconfig
index ca3b02dcbbfac..64b772c5d1c9b 100644
--- a/crypto/Kconfig
+++ b/crypto/Kconfig
@@ -1768,7 +1768,7 @@ config CRYPTO_DRBG_HMAC
 	bool
 	default y
 	select CRYPTO_HMAC
-	select CRYPTO_SHA256
+	select CRYPTO_SHA512
 
 config CRYPTO_DRBG_HASH
 	bool "Enable Hash DRBG"
-- 
cgit 1.2.3-1.el7

